### DISCUSSION TITLE
CuxD Geräte lassen sich nicht einbinden

### DISCUSSION THREAD

**Discussion by ghost** [thkl/hap-homematic#372] [Q&A] [answered]
Hallo, 
die Software ist echt super. Vielen Dank dafür. Ich habe alle meine Homematic Geräte eingebunden bekommen. 

Leider bekomme ich kein virtuell erzeugtes Gerät über CUxD eingebunden. 
Ich habe einen 28iger Gerätetyp als Unterputz Schaltaktor 1-fach erstellt. Das Gerät wird auch in der CCU3 angezeigt und lässt sich dort als Gerät einbinden.

Öffne ich jetzt Hap-Homematic und gehe auf „Geräte“ und „Neu“ finde ich in der Liste das Gerät leider nicht. 

Habe den Vorgang mehrfach wiederholt. Leider ohne Erfolg. Ich möchte über
den virtuellen Schalter einen Shelly1 einbinden und dann über HomeKit schaltbar machen.

Was mache ich falsch oder habe ich etwas vergessen? 

Vielen Dank schonmal für die Antwort.

Schöne Grüße 

Matze

**Comment by blackbasket** [comment, marked as answer]
Hallo Matze,

bei mir werden alle CUxD-Geräte im Plugin angezeigt. Ich habe jetzt kein 28er-Gerät zum sinnvollen Test. Hast du den Cache des Plugins einmal aktualisiert oder auch das Plugin neu gestartet?

LG,
Marcel

**Reply by ghost** [reply]
Hallo Marcel, 

der Neustart des Plug-Ins war die Lösung. Jetzt sehe ich auch den angelegten Schalter über CuXD. 

Dank dir und schönes Wochenende 

Gruß 

Matze

**Reply by blackbasket** [reply]
Gerne :) dir auch
